### ISSUE TITLE
Adding in Azure DevOps (ADO) to our Product Compatibility

### ISSUE THREAD

**Issue by shanecarr1** [sourcegraph/handbook#2381]
Hey team 👋  @benvenker @macraig @ryankscott @malomarrec @Joelkw @ryphil @bcreager @dan-mckean 

Is it possible to add in Azure DevOps (ADO) to our [Product Compatibility page in the handbook](https://handbook.sourcegraph.com/departments/product-engineering/product/tools/feature_compatibility/) ? 

I think that's a pretty big/main code host that should be addressed. During our Q1 QBR's there were a few different reps that had ADO come up and weren't completely sure how Sourcegraph works there.

Thank you for your time with this 😄 

cc
@wcamp123 @danryan621

**Comment by malomarrec** [comment]
Thanks for raising. I don't think we have full support for this, but we have minimal support by [git clone URL](https://docs.sourcegraph.com/admin/external_service/other) like for any git-based code host.
That's not represented here, should we have a note saying that "any other git codehost" has "partial support"?
cc @dan-mckean

**Comment by danryan621** [comment]
thanks for bringing this up.  Adding ADO to that page would be really helpful.  Would this page also be the right place for adding configuration options?  Some of the challenges we had with them (in addition to product compatibility) were in regards to configurable services and if they could use ADO PaaS components

**Comment by malomarrec** [comment]
I don't think this page is the right place for configuration options we don't fully support, because this page is a high-level summary of codehosts for which we have dedicated support. Adding a tick here would not reflect the limitations. @dan-mckean 's call though.

 However, if ADO is something that is frequently requested, and it's unclear how well we support it, how about having an ADO documentation page that explains how-to set it up (using basic git code host support) and outlining limitations?

**Comment by danryan621** [comment]
@malomarrec that makes sense.  An ADO documentation page would be great.  We don't have a ton of clients requesting it but recently we have seen some significant clients bring it up.  I believe it may be more common as we move up market.

**Comment by dan-mckean** [comment]
Sorry - this conversation has only just surfaced for me!

@danryan621 I'm not seeing any product gaps logged for this - and that's how we'd track requests and evaluate the worth of adding it. 

Are you aware of who else is asking for it? Can you create some gap submissions so we can assess it?

(This applies if all we do is add a page to cover existing support, or go all the way to provide full support.)

**Comment by danryan621** [comment]
@dan-mckean sorry for the delayed response.  I hadn't created a product gap because we've already worked through most of the questions that JLL had.  I created one just now for them since this may come up in regards to upgrades as well.  Microsoft also asked about it but I'm not sure of the needs of that opportunity.  Looping in @justdueck for visibility

**Comment by malomarrec** [comment]
From @danryan621, saved here.
>@dan-mckean sorry for the delayed response. I hadn't created a product gap because we've already worked through most of the questions that xxx had. I created one just now for them since this may come up in regards to upgrades as well. yyy also asked about it but I'm not sure of the needs of that opportunity. Looping in @justdueck for visibility


@danryan621 this is a public issue; you may want to remove customer names :). I deleted the message meanwhile.
One way to mention customers publicly is to copy paste links to this repo, eg. https://github.com/sourcegraph/accounts/issues/641, so that someone that is not logged in with a Sourcegraph account cannot see them.
See https://handbook.sourcegraph.com/departments/bizops/tools/customer_ops_tools/#linking-to-customer-or-prospect-names-in-public-places

**Comment by danryan621** [comment]
Thanks @malomarrec!

**Comment by leo-abreu** [comment]
Attached is a message received by an SDR today from someone having issues with configuring a git repo in ADO.
Sometimes, situations like these won't make it to a qualified opportunity where a Product Gap could be logged.
So +1 on a how-to doc on our site.  It may go a long way.

![image](https://user-images.githubusercontent.com/46694888/160011890-deb04cb4-45c3-4037-a4e7-225ef1986fe9.png)

as for the links referenced in the message: It's [this walkthrough](https://medium.com/@r.z./use-sourcegraph-with-azure-devops-repos-ab8921236d19) and [this feature request](https://github.com/sourcegraph/sourcegraph/issues/32717)
